recipe/meta.yaml

```diff
@@ -27,7 +28,9 @@ requirements:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - {{ compiler("cuda") }}
+    - {{ stdlib("c") }}
     - make
+    - patch
```

Comment:
Am curious why `patch` is needed here? AIUI [conda-build already ships with `patch`]( https://github.com/conda-forge/conda-build-feedstock/blob/52941906c19102feaf86271d87afc2f31b87ddc4/recipe/meta.yaml#L48 ). So it will use that `patch` and not the one we install

Unless for some reason we need to apply the patch ourselves without conda-build. If so, would be interested in understanding that context better

```suggestion
```

Reply:
Well, the context is that we put patch in our recipes when we want to patch, although I don't think it's needed (to be explicit maybe)? Will remove